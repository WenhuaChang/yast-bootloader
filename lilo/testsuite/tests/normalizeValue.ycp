{
    include "testsuite.ycp";
    include "lilo/routines.ycp";
    
    TEST(``(normalizeValue("change", "a,b,c,d")), [], nil);
    TEST(``(normalizeValue("change", "change,a,b,c,d")), [], nil);
    TEST(``(normalizeValue("change-rules", "a,b,c,d")), [], nil);
    TEST(``(normalizeValue("change-rules", "")), [], nil);
    TEST(``(normalizeValue("A", "B")), [], nil);
    TEST(``(normalizeValue("A", "")), [], nil);
    TEST(``(normalizeValue("A", 6)), [], nil);

}
