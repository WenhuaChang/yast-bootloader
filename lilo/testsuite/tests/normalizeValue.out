Read	.probe.architecture "i386"
Read	.probe.has_apm true
Read	.probe.has_pcmcia false
Read	.probe.has_smp false
Read	.probe.system []
Read	.target.tmpdir "/tmp"
Read	.target.size "/etc/install.inf" 0
Read	.etc.install_inf.Sourcemounted 0
Read	.etc.install_inf.AutoYaST 0
Read	.etc.install_inf.Demo 0
Read	.etc.install_inf.Textmode 0
Read	.etc.install_inf.Manual 0
Read	.etc.install_inf.Console 0
Read	.etc.install_inf.Braille 0
Read	.etc.install_inf.VNC 0
Read	.etc.install_inf.UseSSH 0
Read	.etc.install_inf.Locale "en_EN"
Read	.etc.install_inf.InstMode "cd"
Read	.sysconfig.console.CONSOLE_FONT "lat2-16.psfu"
Read	.sysconfig.console.CONSOLE_SCREENMAP "trivial"
Read	.sysconfig.console.CONSOLE_UNICODEMAP ""
Read	.sysconfig.console.CONSOLE_MAGIC "(K"
Read	.sysconfig.console.CONSOLE_ENCODING "ISO-8859-2"
Execute	.target.bash "/usr/bin/setfont" 0
Read	.probe.floppy.manual []
Read	.probe.cdrom.manual []
Read	.probe.boot_arch "lilo"
Return	change,a,b,c,d
Return	change,a,b,c,d
Return	change-rules,a,b,c,d
Return	change-rules
Return	B
Return	
Return	6
